website/docs/ide-config.md

```diff
@@ -0,0 +1,62 @@
+# Vscode
+
+> Vscode Extensions
+
+- ESLint
```

Comment:
Why not try this way? https://stackoverflow.com/a/52580725/14728656

Reply:
Or this way https://tech.serhatteker.com/post/2020-05/vscode-extensions-recommended/

Reply:
> Or this way https://tech.serhatteker.com/post/2020-05/vscode-extensions-recommended/

this is my before, 🤣，https://github.com/apache/apisix-website/pull/1109/commits/58fea5dfd567819d85098851ec672b395d8d0936

Reply:
@barchielAbyss yes, and I mentioned it, but probably didn't make it clear

> What I think is that if we maintain, we must include vscode configuration files and related documents at the same time.
>
> _Originally posted by @SkyeYoung in https://github.com/apache/apisix-website/issues/1111#issuecomment-1133825053_